src/Microsoft.Health.Fhir.Liquid.Converter/Filters/StringFilters.cs

```diff
@@ -105,5 +109,108 @@ public static string Base64Decode(string data)
             var bytes = Convert.FromBase64String(data);
             return Encoding.UTF8.GetString(bytes);
         }
+
+        public static string ToXhtml(string xmlString)
+        {
+            XNamespace xhtmlNamespace = "http://www.w3.org/1999/xhtml";
+            XDocument doc;
+            try
+            {
+                doc = XDocument.Parse(xmlString);
+
+                if (doc.Root.Name.LocalName != "div")
+                {
+                    doc = XDocument.Parse($"<div xmlns=\"http://www.w3.org/1999/xhtml\">{xmlString}</div>");
```

Comment:
I'd recommend doing it this way so you don't have to parse the xml a second time.
```suggestion
                    var ns = XNamespace.Get("http://www.w3.org/1999/xhtml");
                    doc = new XDocument(new XElement(ns + "div", doc.Root));
```